rootfs/etc/nginx/lua/util.lua

```diff
@@ -1,6 +1,14 @@
-local string_len = string.len
-local string_sub = string.sub
+local string        = string
+local string_len    = string.len
+local string_sub    = string.sub
 local string_format = string.format
+local pairs         = pairs
+local tonumber      = tonumber
+local ngx           = ngx
```

Comment:
This will require us to adjust some unit tests as they mock some `ngx` APIs before loading the modules: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/ingress-nginx/4591/pull-ingress-nginx-test-lua/1176327555209236481